http_client.go

```diff
@@ -39,7 +39,7 @@ type HTTPClientConfig struct {
 	// FollowRedirects specifies whether the client should follow HTTP 3xx redirects.
 	// The omitempty flag is not set, because it would be hidden from the
 	// marshalled configuration when set to false.
-	FollowRedirects bool `yaml:"follow_redirects"`
+	FollowRedirects bool `yaml:"follow_redirects,omitempty"`
```

Comment:
In original Prometheus config they don't use `omitempry` for that field: https://github.com/prometheus/common/blob/ce7006ede1b27960d267508d7ae0af2f51422610/config/http_config.go#L197

Also what it should mean if this field Is omitted?

Reply:
Also this field has a comment which mentions `omitmepty`

Reply:
that's weird. It makes sense if it's pointer, but it's not.

Reply:
Well, I propose to stay as close to the original Prometheus config, as possible.